pilot/cmd/pilot-agent/app/cmd.go

```diff
@@ -311,6 +317,73 @@ func initProxy(args []string) (*model.Proxy, error) {
 	return proxy, nil
 }
 
+func applyExcludeInterfaces(ifaces []string) []string {
+	// Get list of excluded interfaces from pod annotation
+	annotations, err := bootstrap.ReadPodAnnotations("")
+	if err != nil {
+		log.Infof("Failed to read pod annotations to find exluded interfaces. Continuing without exclusions. Proxy IPAddresses: %v", ifaces)
+		return ifaces
+	}
+	value, ok := annotations[annotation.SidecarTrafficExcludeInterfaces.Name]
+	if !ok {
+		log.Infof("excludeInterfaces annotation not present. Proxy IPAddresses: %v", ifaces)
```

Comment:
I would make this debugf, not relevant for 99.99% of users

Reply:
agreed.